core/templates/dev/head/pages/exploration_editor/exploration_objective_editor_directive.html

```diff
@@ -0,0 +1,27 @@
+<form name="objectiveForm">
+  <p ng-class="{'has-error': objectiveForm.$dirty && (!explorationObjectiveService.displayed || explorationObjectiveService.displayed.length < 15 || explorationObjectiveService.displayed.length > 100)}">
+    <label for="explorationObjective" ng-style="<[::formStyle]>"><[::labelText]></label>
+    <input id="explorationObjective" type="text"
+          class="form-control protractor-test-exploration-objective-input"
+          ng-model="explorationObjectiveService.displayed"
+          ng-blur="onInputFieldBlur()" placeholder="Learn how to..." ng-trim="false">
+    <span class="help-block" style="font-size: smaller">
+      <em>In a complete sentence, tell people what they'll learn from this exploration.</em>
+      <span ng-if="explorationObjectiveService.displayed.length > 0">
+        <em>Characters used: <[explorationObjectiveService.displayed.length]>/100</em>
+      </span>
+    </span>
+  </p>
+</form>
+<style>
+  @media (min-width: 768px) {
+    exploration-objective-editor p {
+      display: table;
+      margin-bottom: 2%;
+      width: 100%;
+    }
+  }
+  .oppia-long-text .help-block {
```

Comment:
Can you prefix this with "exploration-objective-editor" so that it only applies to this directive and doesn't "leak out"?

Reply:
Hey @seanlip I'm not sure as to what you mean by prefixing the CSS rule to just "exploration-objective-editor", since this is also used by the title directive?

Reply:
Oh, then here you should have `exploration-objective-editor .oppia-long-text .help-block` and in the title directive you should have `exploration-title-editor .oppia-long-text .help-block` -- does that help? Unless it's a more general thing that's going to be used by lots of stuff, in which case it probably goes in oppia.css.

Reply:
Done. Thank you Sean, that helped. I had to make a few changes to the CSS rule to make it space equally, while using the prefix of the directives.

Reply:
Looks good. I was unsure at first of whether the rule belongs in the directive file rather than something "higher", since it depends on external context, but I think it's probably fine as is.

Just one nit comment, then please feel free to merge!